pkg/action/testdata/scan_archive

```diff
@@ -1,7 +1,7 @@
 {
     "Files": {
-        "/apko_0.13.2_linux_arm64/apko": {
-            "Path": "testdata/apko_nested.tar.gz ∴ /apko_0.13.2_linux_arm64/apko",
+        "/apko_0.13.2_linux_arm64/apko_0.13.2_linux_arm64/apko": {
```

Comment:
The duplication in this path is correct. Other archives shouldn't exhibit this behavior unless it was also intentional:
```
$ tar -xzvf apko_nested.tar.gz
apko_0.13.2_linux_arm64.tar.gz
$ tar -xzvf apko_0.13.2_linux_arm64.tar.gz
apko_0.13.2_linux_arm64/LICENSE
apko_0.13.2_linux_arm64/apko
```